ui/app/AppLayouts/Onboarding/views/LoginView.qml

```diff
@@ -263,17 +265,32 @@ Item {
                 closePolicy: Popup.CloseOnEscape | Popup.CloseOnPressOutsideParent
                 width: parent.width + Style.current.bigPadding
                 dim: false
-                Repeater {
+
+                SortFilterProxyModel {
+                    id: proxyModel
+                    sourceModel: root.startupStore.startupModuleInst.loginAccountsModel
+                    filters: ValueFilter {
+                        roleName: "keyUid"
+                        value: root.startupStore.selectedLoginAccount.keyUid
+                        inverted: true
+                    }
+                }
+
+                ListView {
                     id: accounts
-                    model: root.startupStore.startupModuleInst.loginAccountsModel
+                    height: 64 * proxyModel.count
+
+                    model: proxyModel
+
                     delegate: AccountMenuItemPanel {
                         label: model.username
                         image: model.thumbnailImage
                         colorId: model.colorId
                         colorHash: model.colorHash
                         keycardCreatedAccount: model.keycardCreatedAccount
                         onClicked: {
-                            root.startupStore.setSelectedLoginAccountByIndex(index)
+                            var realIndex = proxyModel.mapToSource(index);
```

Comment:
please `var` -> `const`, `;` not needed for consistency

Reply:
Ideally it would be nice to switch to selecting by some unique id (keyUid?). When adding some additional proxies (e.g. for sorting) than we would need to add additional `mapToSource` here. It's easy to miss sth like that. But probably it's out of scope for that PR.